src/cudamatrix/cu-sparse-matrix.cc

```diff
@@ -388,7 +388,7 @@ void CuSparseMatrix<Real>::CopyFromSmat(const CuSparseMatrix<Real>& smat,
       Resize(smat.NumCols(), smat.NumRows(), smat.NumElements(), kUndefined);
       CuTimer tim;
 
-      CUSPARSE_SAFE_CALL(
+      CU_SAFE_CALL(
```

Comment:
Looks to me like there was a merge problem here.  It should be CUSPARSE_SAFE_CALL

Reply:
Fixed it, thanks.